Extension/src/LanguageServer/settings.ts

```diff
@@ -203,6 +226,7 @@ export class CppSettings extends Settings {
     public get clangFormatSortIncludes(): string | undefined { return super.Section.get<string>("clang_format_sortIncludes"); }
     public get experimentalFeatures(): string | undefined { return super.Section.get<string>("experimentalFeatures"); }
     public get suggestSnippets(): boolean | undefined { return super.Section.get<boolean>("suggestSnippets"); }
+    public get caseSensitiveFileSupport(): string | undefined { return super.Section.get<string>("caseSensitiveFileSupport"); }
```

Comment:
Probably want to put a comment here stating that the value could be incorrect if it was set in a workspace folder.

Reply:
Not relevant due to scope change